veg2hab/io/common.py

```diff
@@ -152,15 +152,20 @@ class Veg2HabConfig(BaseSettings):
     class Config:
         env_prefix = "VEG2HAB_"
 
-    mozaiek_threshold: NumberType = Field(
+    combineer_karteringen_weglaten_threshold: float = Field(
+        default=0.0001,
+        description="Threshold in m^2 voor het weglaten van vlakken na het combineren van karteringen",
+    )
+
+    mozaiek_threshold: float = Field(
         default=95.0,
         description="Threshold voor het bepalen of een vlak in het mozaiek ligt",
     )
-    mozaiek_als_rand_threshold: NumberType = Field(
+    mozaiek_als_rand_threshold: float = Field(
```

Comment:
aangezien Numbertype = Union[int, float] dus niet werkt zoals ik dacht (cast als het kan altijd naar int, ook als je een float aanlevert) heb ik er maar overal float van gemaakt, dan is het in de toekomst ieg geen probleem.

Reply:
Ja lijkt me goed.